src/odemis/util/test/img_test.py

```diff
@@ -848,7 +848,7 @@ def getSubData(dast, zoom, rect):
             model.MD_POS: POS,
             model.MD_PIXEL_SIZE: (1e-6, 1e-6),
         }
-        arr = numpy.array(range(size[0] * size[1])).reshape(size[::-1]).astype(dtype)
+        arr = numpy.array(list(range(size[0] * size[1]))).reshape(size[::-1]).astype(dtype)
```

Comment:
In this file you should be able to also use `numpy.arange()` everywhere affected. BTW, this also allows to merge the dtype, like this:
`numpy.arange(size[0] * size[1], dtype=dtype).reshape(size[::-1])`